docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -38,19 +38,19 @@ Some device manufacturers keep Private DNS settings hidden and set 'Automatic' m
 
 To be able to filter HTTPS traffic, AdGuard requires the user to add a certificate to the device's trusted user certificates. Unfortunately, UC-family browsers don't trust user certificates, so AdGuard cannot perform HTTPS filtering there.
 
-- To solve this problem, move the certificate to the system certificate store
+- To solve this problem, move the [certificate to the system certificate store](../solving-problems/https-certificate-for-rooted.md/)
 
-:::caution
+:::note Compatibility
 
 Requires root access.
 
 :::
 
 ### Dolphin Browser: Dolphin Browser, Dolphin Browser Express
 
-AdGuard cannot filter its traffic when operating in **Manual proxy mode** because this browser ignores system proxy settings.
+AdGuard cannot filter its traffic when operating in the *Manual proxy mode* because this browser ignores system proxy settings.
```

Comment:
да блиин)
```suggestion
AdGuard cannot filter its traffic when operating in the *Manual proxy* mode because this browser ignores system proxy settings.
```